src/test/ui/async-await/in-trait/async-generics-and-bounds.rs

```diff
@@ -0,0 +1,21 @@
+// edition: 2021
```

Comment:
This should have a bug filed for it

Reply:
Filed https://github.com/rust-lang/rust/issues/102682